packages/kbn-health-gateway-server/src/index.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { ServiceConfigDescriptor } from '@kbn/core-base-server-internal';
+import {
+  config as loggingConfig,
+  LoggingSystem,
+  LoggingConfigType,
+} from '@kbn/core-logging-server-internal';
+import { getConfigService } from './config';
+import { config as kibanaConfig, KibanaService } from './kibana';
+import { config as serverConfig, Server } from './server';
+
+export async function bootstrap() {
+  const loggingSystem = new LoggingSystem();
+  const logger = loggingSystem.asLoggerFactory();
+  const configService = getConfigService({ logger });
+
+  const configDescriptors: ServiceConfigDescriptor[] = [loggingConfig, kibanaConfig, serverConfig];
+  for (const { path, schema } of configDescriptors) {
+    configService.setSchema(path, schema);
+  }
+
+  await configService.validate();
+
+  await loggingSystem.upgrade(configService.atPathSync<LoggingConfigType>('logging'));
+  const log = logger.get('root');
+
+  const server = new Server({ config: configService, logger });
+  const serverStart = await server.start();
+
+  const kibanaService = new KibanaService({ config: configService, logger });
+  await kibanaService.start({ server: serverStart });
```

Comment:
I think it's worth wrapping in a try catch and logging with the logger so that we can see an exception here in our cloud logs